### ISSUE TITLE
Logo

### ISSUE THREAD

**Issue by XhmikosR** [Johann-S/bs-customizer#16]
This is minor of course but we should have a logo so that we add favicons etc

**Comment by Johann-S** [comment]
That's so true !

I would love to have a logo ❤️

**Comment by richardbmx** [comment]
@XhmikosR  Hi, my name is Richard. I'm a graphic designer and I love making contributions for open source projects. I see that they want a new logo for your application. If you wish, I can do it for you without any problem ... I can present some modern ideas that would improve the appearance and image of your project

**Comment by Johann-S** [comment]
Hi Richard, it would be a pleasure to see your creations ! 👍

**Comment by richardbmx** [comment]
@Johann-S  perfect, can you give me an explanation of what the project is about

**Comment by Johann-S** [comment]
Sure, `bs-customizer` allow you to build a custom version of Bootstrap with just what you need.

As you can see here: https://getbootstrap.com/docs/4.1/getting-started/introduction/ Bootstrap provide a lot of things for developers, so it's pretty convenient to just pick what you want from Bootstrap

**Comment by richardbmx** [comment]
@Johann-S  Do you have any idea what would be the specific figure that represents your project with ease? Or would you like a design for example of the letter B in a minimalist way and that is your icon for being the initial of the name of your project?

**Comment by Johann-S** [comment]
Maybe you'll think I'm difficult, but I would like something which looks like the Bootstrap Logo, but a bit different to be easely recognized 😆 

The Bootstrap logo: https://getbootstrap.com/docs/4.1/assets/img/bootstrap-stack.png

**Comment by richardbmx** [comment]
@Johann-S  Perfect I will work on it

**Comment by XhmikosR** [comment]
Maybe a cog along with the Bootstrap logo?

**Comment by MartijnCuppens** [comment]
@richardbmx, the original Bootstrap logo can be downloaded here: https://getbootstrap.com/docs/4.1/about/brand/#download-mark

**Comment by richardbmx** [comment]
I present this first idea I am working on other models

![logo106 1](https://user-images.githubusercontent.com/35604837/46172844-8815f200-c272-11e8-8ff7-85ffd7ae16d9.png)

**Comment by Johann-S** [comment]
I like that one : 
![image](https://user-images.githubusercontent.com/1689750/46173576-e957b880-c2a6-11e8-9abb-b1fbb6f26595.png)

But I can't wait to see your other models 👍

**Comment by richardbmx** [comment]
Second model is a more specific design incorporates the letter b in the center of a circle of different layers to give the library function

![logo106 2](https://user-images.githubusercontent.com/35604837/46184076-f02ffc80-c2a1-11e8-9404-6663330bf2cf.png)

**Comment by richardbmx** [comment]
This terser model is a bit more work on the letter b giving a modern and minimalist style that looks very good

![logo106 3](https://user-images.githubusercontent.com/35604837/46186630-95040700-c2ad-11e8-8e44-ea724ff1f201.png)

**Comment by Johann-S** [comment]
Wow you did an awesome work @richardbmx thank you !

Finally I prefer the last one in grey 😍
Which one do you prefer @XhmikosR and @MartijnCuppens ?

**Comment by MartijnCuppens** [comment]
Nice logo's indeed!

But I'm missing the link with Bootstrap a bit, it looks more like a totally new kind of thing. And because you changed the font, I would think BS customiser also adds a theming layer on top of Bootstrap. 

Can you maybe make some variations with the Bootstrap logo, maybe with a cog as @XhmikosR suggested?

**Comment by XhmikosR** [comment]
Yeah, I don't think we need a completely new logo :) We just need the official logo with a cog or something to indicate the customizer part.

**Comment by Johann-S** [comment]
I think we can have both and we're not part of Bootstrap (here).

So this logo: 

![image](https://user-images.githubusercontent.com/1689750/46199535-97974880-c30f-11e8-9b6e-32a0f8a176f2.png)

But with a cog.

Is it ok for you ?

**Comment by XhmikosR** [comment]
We are not part of Bootstrap isn't true ;)

Plus the plan is to make this official later.

**Comment by richardbmx** [comment]
@Johann-S  perfect to use the same Bootstrap logo you need to have the copyright of that logo with permission to modify it that's why I did not use it from the beginning

**Comment by richardbmx** [comment]
Perfect is this what you want?

![bootstrap-solid2](https://user-images.githubusercontent.com/35604837/46209288-7cbad900-c2fb-11e8-8e09-890d42ae215d.png)

**Comment by Johann-S** [comment]
@richardbmx it looks good to me 👍 

Any feedbacks ? 

/CC @XhmikosR or @MartijnCuppens

**Comment by XhmikosR** [comment]
Only thing I'd change is rotate the cog like 1 degree to the right so that it doesn't go below the horizontal line.

![46209338-9bb96b00-c2fb-11e8-8669-18795ece912b](https://user-images.githubusercontent.com/349621/46284603-1e9f2780-c581-11e8-9e9f-726727cf25f8.png)

**Comment by MartijnCuppens** [comment]
I think the cog also covers a tiny bit of the B (see golden arrow):  

![image](https://user-images.githubusercontent.com/11559216/46303214-8702fe80-c5ab-11e8-952d-e31178b47315.png)

**Comment by Johann-S** [comment]
I like that the alternative, it looks like Bootstrap but it shows it's a different project 👍

**Comment by Johann-S** [comment]
@XhmikosR and @MartijnCuppens which one do you want ?

Personnaly the second one 👍

**Comment by XhmikosR** [comment]
The ultimate choice is yours, @Johann-S :) That being said the second one does seem better.

**Comment by MartijnCuppens** [comment]
What @XhmikosR said 😉

**Comment by richardbmx** [comment]
I am waiting for you to confirm the definitive one to generate the final folder with all the necessary files to use it. Greetings. ;-)

**Comment by Johann-S** [comment]
It seems we all agree one the second one @richardbmx 👍

**Comment by richardbmx** [comment]
@Johann-S  this is the definitive ??

![2222](https://user-images.githubusercontent.com/35604837/46538684-95d70300-c882-11e8-82b7-e556f5bf73cd.png)

**Comment by Johann-S** [comment]
exactly 👍

**Comment by richardbmx** [comment]
@Johann-S  Here I give you the link with all the necessary files to use the logo, inside you find different variations of the logo to be used in different applications as well as the original files to edit them if you need it. Any questions you can write me I am ready for next projects if you wish...

[Google Drive](https://drive.google.com/open?id=1WBCQ6rjAhTsQZ75AvLUa9hTJr7FG8gPn)

**Comment by Johann-S** [comment]
Hi @richardbmx thank you for your hardwork ! 👍 

I have some little projects but I don't think they will be used a lot, but I'll keep you in touch if I need a logo 👍

**Comment by richardbmx** [comment]
@Johann-S  Perfect it was a pleasure to contribute in your project we will be in contact
